manim/mobject/mobject.py

```diff
@@ -1,3 +1,7 @@
+from __future__ import annotations
+
+from _typeshed import SupportsLessThan
```

Comment:
Probably should remove this actually. Rather unfortunate that Python doesn't care enough about type correctness to put type annotations on the standard library.